src/klein/_app.py

```diff
@@ -366,20 +432,24 @@ def urlFor(
                     " doesn't contain Host header"
                 )
             host = b""
-        return self.url_map.bind(host).build(
-            endpoint, values, method, force_external, append_unknown
+        return cast(
+            Text,
+            self.url_map.bind(host).build(
+                endpoint, values, method, force_external, append_unknown
+            ),
         )
```

Comment:
This looks like an almost exact copy of the call to `build` above, with the same call to `cast`.  Is there any way to factor these together so the `cast` only has to be explained once?

Reply:
4798fdd2e9d7397f2be8fb90f55779b52438f677